packages/orchestration/src/exos/orchestrator.js

```diff
@@ -36,32 +34,23 @@ const trace = makeTracer('Orchestrator');
 /** @see {Orchestrator} */
 export const OrchestratorI = M.interface('Orchestrator', {
   getChain: M.call(M.string()).returns(Vow$(ChainInfoShape)),
-  makeLocalAccount: M.call().returns(Vow$(LocalChainAccountShape)),
   getDenomInfo: M.call(DenomShape).returns(DenomInfoShape),
   asAmount: M.call(DenomAmountShape).returns(AmountShape),
 });
 
 /**
  * @param {Zone} zone
  * @param {{
- *   asyncFlowTools: AsyncFlowTools;
  *   chainHub: ChainHub;
- *   localchain: Remote<LocalChain>;
- *   makeRecorderKit: MakeRecorderKit;
```

Comment:
I expect some of these will need to be restored later but I agree it's better to leave them out until they're used